<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegdConverterWindow</class>
 <widget class="QMainWindow" name="SegdConverterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SegdConverterWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/imgs/geophis.png</normaloff>:/icons/imgs/geophis.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="mainPreferencesGroupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="groupBoxVerticalLayout">
         <item>
          <layout class="QHBoxLayout" name="segdHorizontalLayout">
           <item>
            <widget class="QLabel" name="segdLabel">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Файл SEGD</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buddy">
              <cstring>segdLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="segdLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="segdPushButton">
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="outFileHorizontalLayout">
           <item>
            <widget class="QLabel" name="outFileLabel">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Файл СST</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buddy">
              <cstring>outFileLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="outFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="outFilePushButton">
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="attrFilehorizontalLayout">
           <item>
            <widget class="QLabel" name="attrFIleLabel">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Файл Атрибутов</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buddy">
              <cstring>attrFileLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="attrFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="attrFilePushButton">
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="logTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="filterGroupBox">
          <property name="title">
           <string>Параметры фильтрации</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QComboBox" name="firstFilterComboBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>От</string>
             </property>
             <property name="buddy">
              <cstring>firstFilterFromLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="firstFilterFromLineEdit"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>До</string>
             </property>
             <property name="buddy">
              <cstring>firstFilterToLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLineEdit" name="firstFilterToLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="secondFilterComboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>От</string>
             </property>
             <property name="buddy">
              <cstring>secondFilterFromLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="secondFilterFromLineEdit"/>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>До</string>
             </property>
             <property name="buddy">
              <cstring>secondFilterToLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="secondFilterToLineEdit"/>
           </item>
           <item row="1" column="5">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>628</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QComboBox" name="thirdFilterComboBox"/>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>От</string>
             </property>
             <property name="buddy">
              <cstring>thirdFilterFromLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="thirdFilterFromLineEdit"/>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>До</string>
             </property>
             <property name="buddy">
              <cstring>thirdFilterToLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLineEdit" name="thirdFilterToLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="attributesTableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenSegd"/>
    <addaction name="actionSaveSeisData"/>
    <addaction name="actionSaveAttributes"/>
    <addaction name="actionSaveAsAttributes"/>
    <addaction name="separator"/>
    <addaction name="actionOpenRPS"/>
    <addaction name="actionOpenSPS"/>
    <addaction name="actionOpenXPS"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionBinPreferences"/>
    <addaction name="actionAttributesPreferences"/>
    <addaction name="actionConvertParameters"/>
    <addaction name="actionAuxes"/>
    <addaction name="separator"/>
    <addaction name="actionAttributeTable"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenSegd"/>
   <addaction name="actionSaveSeisData"/>
   <addaction name="actionSaveAttributes"/>
   <addaction name="actionSaveAsAttributes"/>
   <addaction name="separator"/>
   <addaction name="actionOpenRPS"/>
   <addaction name="actionOpenSPS"/>
   <addaction name="actionOpenXPS"/>
   <addaction name="separator"/>
   <addaction name="actionBinPreferences"/>
   <addaction name="actionAttributesPreferences"/>
   <addaction name="actionConvertParameters"/>
   <addaction name="separator"/>
   <addaction name="actionAttributeTable"/>
   <addaction name="actionFilter"/>
   <addaction name="actionSorting"/>
   <addaction name="actionErase"/>
   <addaction name="separator"/>
   <addaction name="actionAuxes"/>
   <addaction name="actionAuxesDisplay"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenSegd">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/sine-waves-analysis.png</normaloff>:/icons/imgs/sine-waves-analysis.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл Segd</string>
   </property>
  </action>
  <action name="actionSaveSeisData">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/save.png</normaloff>:/icons/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать файл CST</string>
   </property>
  </action>
  <action name="actionSaveAttributes">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/xlsx.png</normaloff>:/icons/imgs/xlsx.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать файл атрибутов</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/delete.png</normaloff>:/icons/imgs/delete.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionBinPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/3d.png</normaloff>:/icons/imgs/3d.png</iconset>
   </property>
   <property name="text">
    <string>Параметры бинирования</string>
   </property>
  </action>
  <action name="actionAttributesPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/calculator.png</normaloff>:/icons/imgs/calculator.png</iconset>
   </property>
   <property name="text">
    <string>Параметры окон расчета</string>
   </property>
  </action>
  <action name="actionOpenSPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/tool.png</normaloff>:/icons/imgs/tool.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать файл SPS</string>
   </property>
  </action>
  <action name="actionOpenRPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/key.png</normaloff>:/icons/imgs/key.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать файл RPS</string>
   </property>
  </action>
  <action name="actionConvertParameters">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/settings-1.png</normaloff>:/icons/imgs/settings-1.png</iconset>
   </property>
   <property name="text">
    <string>Параметры конвертации</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/circle.png</normaloff>:/icons/imgs/circle.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/black.png</normaloff>:/icons/imgs/black.png</iconset>
   </property>
   <property name="text">
    <string>О Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/play.png</normaloff>:/icons/imgs/play.png</iconset>
   </property>
   <property name="text">
    <string>Запуск конвертации</string>
   </property>
   <property name="toolTip">
    <string>Запуск конвертации</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/square-1.png</normaloff>:/icons/imgs/square-1.png</iconset>
   </property>
   <property name="text">
    <string>Остановка Конвертации</string>
   </property>
   <property name="toolTip">
    <string>Остановка Конвертации</string>
   </property>
  </action>
  <action name="actionAttributeTable">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/document-with-table.png</normaloff>:/icons/imgs/document-with-table.png</iconset>
   </property>
   <property name="text">
    <string>Настройка таблицы атрибутов</string>
   </property>
  </action>
  <action name="actionOpenXPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/x_letter.png</normaloff>:/icons/imgs/x_letter.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать файл XPS</string>
   </property>
  </action>
  <action name="actionAuxes">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/wave.png</normaloff>:/icons/imgs/wave.png</iconset>
   </property>
   <property name="text">
    <string>Настройки служебных каналов</string>
   </property>
  </action>
  <action name="actionSaveAsAttributes">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/xlsxSaveAs.png</normaloff>:/icons/imgs/xlsxSaveAs.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить атрибуты как...</string>
   </property>
   <property name="toolTip">
    <string>Сохранить атрибуты как...</string>
   </property>
  </action>
  <action name="actionAuxesDisplay">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/waveDisplay.png</normaloff>:/icons/imgs/waveDisplay.png</iconset>
   </property>
   <property name="text">
    <string>Служебные каналы</string>
   </property>
   <property name="toolTip">
    <string>Отобразить окно служебных каналов</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/filter.png</normaloff>:/icons/imgs/filter.png</iconset>
   </property>
   <property name="text">
    <string>Фильтрация атрибутов</string>
   </property>
   <property name="toolTip">
    <string>Отобразить параметры фильтрации атрибутов</string>
   </property>
  </action>
  <action name="actionSorting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/sort-by-numeric-order.png</normaloff>:/icons/imgs/sort-by-numeric-order.png</iconset>
   </property>
   <property name="text">
    <string>Включить Сортировку</string>
   </property>
   <property name="toolTip">
    <string>Включить Сортировку</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/eraser.png</normaloff>:/icons/imgs/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Очистить данные</string>
   </property>
   <property name="toolTip">
    <string>Очистить таблицу рассчитанных атрибутов и служебные трассы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionConvertParameters</sender>
   <signal>triggered()</signal>
   <receiver>SegdConverterWindow</receiver>
   <slot>openConvertParametersDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SegdConverterWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBinPreferences</sender>
   <signal>triggered()</signal>
   <receiver>SegdConverterWindow</receiver>
   <slot>openBinningParametersDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAttributesPreferences</sender>
   <signal>triggered()</signal>
   <receiver>SegdConverterWindow</receiver>
   <slot>openWindowsParametersDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openConvertParametersDialog()</slot>
  <slot>openBinningParametersDialog()</slot>
  <slot>openWindowsParametersDialog()</slot>
 </slots>
</ui>
